src/Microsoft.Health.Fhir.ValueSets/FhirAnonymousOperationType.cs

```diff
@@ -0,0 +1,16 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+namespace Microsoft.Health.Fhir.ValueSets
+{
+    /// <summary>
+    /// Value set for Fhir operations which do not require authorization
+    /// </summary>
+    public static class FhirAnonymousOperationType
```

Comment:
This doesn't look like a FHIR ValueSet

Reply:
hm right, moved to Fhir.Api.Features.AnonymousOperations. Let me know if there is a better place